test-cases/longevity/longevity-100gb-4h.yaml

```diff
@@ -6,15 +6,19 @@ stress_cmd: ["cassandra-stress write cl=QUORUM duration=240m -schema 'replicatio
              "cassandra-stress read  cl=QUORUM duration=240m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=40 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
              ]
 
-n_db_nodes: 6
-n_loaders: 2
+n_db_nodes: 5
+n_db_zero_token_nodes: 1
+
+n_loaders: 1
 n_monitor_nodes: 1
 seeds_num: 3
+num_zero_token_nodes: 1
+zero_token_instance_type_db: 'i4i.xlarge'
```

Comment:
I suggest to make it something tiny, as any user would do.
it shouldn't even be i4i, could be something that doesn't have NVMe or has very small NVMe.

It probably raises a requirement for scylla-machine-image to support such instance type for zero-token-node (shouldn't block the installation).